trame_client/ui/core.py

```diff
@@ -66,3 +98,35 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._original_root.__exit__(exc_type, exc_value, exc_traceback)
         self.flush_content()
+
+    # -------------------------------------------------------------------------
+    # Jupyter display integration
+    # -------------------------------------------------------------------------
+
+    def _jupyter_content(self):
+        if not self.server.running:
+            return """<div>
+                    The server is not running. Before displaying a trame layout you should await it to make sure it is ready.
+                    You should run the following code before displaying a layout.
+                    <pre style="padding: 5px; border: solid 1px rgb(224,224,224); background: rgb(245,245,245);">await layout.ready</pre>
+                    </div>"""
+
+        elem_id = f"{self.server.name}_{self._template_name}"
+        src = f"{self.iframe_base_url}:{self.server.port}/index.html?ui={self._template_name[16:]}"
+        return (
+            f'<iframe id="{elem_id}" src="{src}" style="{self.iframe_style}"></iframe>'
+        )
+
+    @property
+    def widget(self):
```

Comment:
@banesullivan I'm just worried about that name. Should this be `ipywidget` instead to make it explicit?